includes/class-wc-stripe-settings.php

```
@@ -0,0 +1,564 @@
+		return $this->settings['pmc_enabled'] ?? '';
+	}
+
+	/**
+	 * Set the `pmc_enabled` setting.
+	 *
+	 * @param string $value The value to set.
+	 */
+	public function set_pmc_enabled( string $value ): void {
+		$this->settings['pmc_enabled'] = $value;
+		$this->save();
+	}
```

PHPMD (2.15.0)

[error] The method set_pmc_enabled is not named in camelCase. (undefined)

(CamelCaseMethodName)